src/app/pages/client/explore/Explore.tsx

```diff
@@ -131,7 +130,7 @@ export function AddServer() {
                     <Text size="B400">Save</Text>
                   </Button> */}
 
-                  <Button type="submit" onClick={handleView} variant="Secondary" fill="Soft">
+                  <Button type="submit" onClick={handleSubmit} variant="Secondary" fill="Soft">
```

Comment:
Yes, we should also add view button for viewing without saving the server in list.